src/JestTestRunner.ts

```diff
@@ -0,0 +1,129 @@
+import { RunnerOptions, RunResult, RunStatus, TestResult, TestRunner, TestStatus } from 'stryker-api/test_runner';
+import { EventEmitter } from 'events';
+
+import * as _ from 'lodash';
+const os = require('os');
+const path = require('path');
+
+const Console = require('console').Console;
+const Runtime = require('jest-runtime');
+
+const runTest = require('jest-cli/build/runTest');
+
+import * as log4js from 'log4js';
+const log = log4js.getLogger('JestTestRunner');
+
+const DEFAULT_OPTIONS: Object = {
+  cacheDirectory: path.join(os.tmpdir(), 'jest'),
+  collectCoverage: false,
+  haste: {},
+  maxWorkers: 1,
+  moduleFileExtensions: ['js', 'json', 'jsx', 'node'],
+  moduleNameMapper: [],
+  setupFiles: [],
+  snapshotSerializers: [],
+  testEnvironment: 'jest-environment-jsdom',
+  testRegex: '.*Spec\\.js$',
+  testRunner: 'jest-jasmine2',
+  verbose: true
+};
+
+export default class JestTestRunner extends EventEmitter implements TestRunner {
+  // Seems there are no up-to-date TypeScript definitions for Jest
+
+  private hasteContext: Promise<any>;
+  private options: any;
+  private paths: string[];
+
+  constructor(options: RunnerOptions) {
+    super();
+    log.debug(`Received options ${JSON.stringify(options)}`);
+
+    this.options = _.assign(DEFAULT_OPTIONS, {
+      rootDir: process.cwd()
```

Comment:
Did you by any chance forgot to merge in the `feature/runner` branch of stryker-mutator itself? I made a windows fix: https://github.com/stryker-mutator/stryker-jest-runner/blob/feature/runner/src/JestTestRunner.ts#L44

 https://github.com/stryker-mutator/stryker-jest-runner/tree/feature/runner